ios/FluentUI/Vnext/IndeterminateProgressBar/IndeterminateProgressBarModifiers.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+public extension IndeterminateProgressBar {
```

Comment:
can you remind me why we do this public extension in a separate file?

Reply:
It's just a standard we're following for the sake of organizing the modifiers for a specific SwiftUI control into its own file. All of the controls with SwiftUI support have it in a separate file.